agens-no/iMessageStickerUnity
# Resolve "Sorting order of sticker sequence is not matching StickerPack.asset"

### Summary
Fix stickers order in Messages app being different from order in StickerPack.asset.

### Details
For sequence type stickers set extension to be `.stickersequence` instead of `.sticker` as mentioned in the Apple docs [here](https://developer.apple.com/library/archive/documentation/Xcode/Reference/xcode_ref-Asset_Catalog_Format/StickerFilmStrip.html#//apple_ref/doc/uid/TP40015170-CH53-SW1).

Relevant commits:
* 45566c2
* 0343b7e
* b689cf1

Bonus fixes:
* Removed missing scene reference from build settings (as it's still displayed in the window): 71c5a9e
* Format all non-XcodeAPI code (decided not to touch that as it's mostly from bitbucket) using VS Code C# extension formatter: 4a52ef7
* Remove Unity services project id as it triggers warning popup when building the project: 83a265b

### How to test
Build the project, observe stickers order follow order in the asset.

Tested with Xcode 11.3, iPhone 8 (11.4) simulator.

<details><summary>Screenshots</summary>
<p>

**Current master build**
![Simulator Screen Shot - iPhone 8 - 2020-03-26 at 21 48 01](https://user-images.githubusercontent.com/5451929/77657340-20984a00-6fb0-11ea-9bec-a9eacbe2fde1.png)

**Pull request branch build**
![Simulator Screen Shot - iPhone 8 - 2020-03-26 at 21 50 36](https://user-images.githubusercontent.com/5451929/77657461-4b829e00-6fb0-11ea-9177-101e710dcfa3.png)

</p>
</details>

Closes #43
